src/Security/ApiKeyAuthenticator.php

```diff
@@ -35,12 +38,18 @@ public function createToken(Request $request, $providerKey)
         );
     }
 
-    public function supportsToken(TokenInterface $token, $providerKey)
+    /**
+     * @param string $providerKey
+     */
+    public function supportsToken(TokenInterface $token, $providerKey): bool
```

Comment:
The ApiKeyAuthenticator is still extending [SimpleAuthenticatorInterface](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Security/Core/Authentication/SimpleAuthenticatorInterface.php)

Deprecated and we should use guard, but for now we can not say: string $providerKey because it conflicts with the interface.